### PR TITLE
Check circles membership when trying to add

### PR DESCRIPTION
**Pull request by nickvergessen**
Signed-off-by: Joas Schilling <coding@schilljs.com>

### CODE REVIEW

File: lib/Service/ParticipantService.php [hunk 1]
```diff
@@ -396,6 +397,30 @@ public function addGroup(Room $room, IGroup $group, array $existingParticipants
...
+			$circle = Circles::detailsCircle($circleId);
+		} catch (\Exception $e) {
+			throw new ParticipantNotFoundException('Circle not found');
+		}
+
+		// FIXME use \OCA\Circles\Manager::getLink() in the future
+		$membersInCircle = $circle->getInheritedMembers();
+		foreach ($membersInCircle as $member) {
+			if ($member->getUserType() === Member::TYPE_USER && $member->getUserId() === $userId) {
+				return $circle;
```
Comment (ArtificialOwl) [inline lib/Service/ParticipantService.php:417]: Overkill: check also that `$member->isLocal()`

**Review by ArtificialOwl — APPROVED** [review]

File: lib/Service/ParticipantService.php (hunk 1 above)
Comment (nickvergessen) [reply lib/Service/ParticipantService.php:417]: getUserType not covering this?

Comment (ArtificialOwl) [reply lib/Service/ParticipantService.php:417]: in GlobalScale setup, userId might be identical while having different `getIinstance()`. isLocal() confirm the user is local

**Pull request merged**